src/Card/Card.js

```diff
@@ -1,16 +1,21 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import Login from '../Login/Login.js';
+import ShowPage from '../ShowPage/ShowPage';
+import { Route, Switch, Link, NavLink } from 'react-router-dom';
 import './Card.css';
 
 const Card = (props) => {
   return (
     <div id={props.id} className='poster-card'>
-      <button>
+      <nav>
+      <Link to={'/movies/' + props.id}>
         <img className='poster' src={props.posterPath} alt= 'movie poster'/>
-        <article>
-          <h2>{props.averageRating}</h2>
-        </article>
-      </button>
+      </Link>
```

Comment:
I think it was good that we moved the rating out of the link tag, but we might ultimately want the entire card to be clickable. Let's revisit this after we do styling(css).